Freetime.xcodeproj/project.pbxproj

```diff
@@ -2025,7 +2025,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\n  swiftlint\nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi";
+			shellScript = "\"${PODS_ROOT}/SwiftLint/swiftlint\"";
```

Comment:
Oh and this had been mildly infuriating me for ages 😂 It keeps trying to use my local version of SwiftLint so I get about 300 violations and can't build anything! This uses the actual version from Pods